docs/conf.py

```diff
@@ -0,0 +1,105 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- Project information -----------------------------------------------------
+
+copyright = '2020-present, Inrupt Inc.'
+
+# -- General configuration ---------------------------------------------------
+
+
+# -- product name -----
+# -- Separately update code samples and toc links and docs-navbar since not using substitutions--
+
+name = 'solid-client'
+repo_name = '{0}-js'.format(name)
+replacement_string = '.. |product|  replace:: ``{0}``'.format(name)
+
+rst_epilog = '\n'.join([
+    replacement_string,
+])
+
+pygments_style = 'sphinx'
+
+# -- Using .txt instead of .rst for restructured text -----------------------
+source_suffix = {
+    '.txt': 'restructuredtext',
```

Comment:
I was playing around with this branch a bit and noticed this. Would you mind too much if we just used `.rst` for ReST? Then I can set up my editor to do automatic syntax highlighting and such for ReST files :)

Reply:
that's fine.

Reply:
Updated to use .rst